packages/react-component-ref/src/Ref.tsx

```diff
@@ -14,7 +14,7 @@ export interface RefProps {
    *
    * @param {HTMLElement} node - Referred node.
    */
-  innerRef: React.Ref<any>
```

Comment:
This change is not correct. `innerRef` will be passed to `RefForwardRef` and there are no guarantees that it will be `HTMLElement`

Reply:
oops, reverted, but the doc comment above is misleading 😕